src/swirl.jl

```diff
@@ -0,0 +1,40 @@
+"""
+    swirl(img, rotation, strength, radius, x0 = OffsetArrays.center(img))
+
+Create whirlpool effect on `img`
+
+# Arguments
+- `img`: to specify input img array
+- `rotation`: to specify rotation angle
+- `strength`:to specify amount of swirl 
+- `radius`:to specify extent of swirl 
+- `x0`: to specify center of swirl.
+
+# Examples
+
+```julia
+using ImageTransformations
+using OffsetArrays, StaticArrays
+using ImageShow, TestImages
+using LinearAlgebra
```

Comment:
Only give what's necessary here:

```suggestion
using ImageTransformations, TestImages
```